hermes/probe/read/read_test.go

```diff
@@ -65,16 +78,44 @@ func TestReadFile(t *testing.T) {
 		TotalSpaceAllocatedMib: int64(100),
```

Comment:
If our filesize is 100 bytes, why are we allocating 100MiB?

Reply:
Thank you for catching that! I chaned to 1MiB I miscalculated....